src/core/jupyter/jupyter.ts

```diff
@@ -1909,11 +1909,14 @@ function mdImageOutput(
   const metadata = output.metadata[mimeType];
 
   // attributes (e.g. width/height/alt)
-  function metadataValue<T>(key: string, defaultValue: T) {
-    return metadata && metadata[key] ? metadata["key"] as T : defaultValue;
+  function metadataValue<T>(key: string, alt_key: string, defaultValue: T) {
+    if (metadata) {
+      return metadata[key] ? metadata[key] as T : metadata[alt_key] ? metadata[alt_key] as T : defaultValue;
+    }
+    return defaultValue;
   }
-  let width = metadataValue(kCellOutWidth, 0);
-  let height = metadataValue(kCellOutHeight, 0);
+  let width = metadataValue(kCellOutWidth, "width", 0);
+  let height = metadataValue(kCellOutHeight, "height", 0);
```

Comment:
I couldn't think of a case were the `metadata` would contain `KCellOutWidth="out-width"` & `KCellOutHeight="out-height"`. Is this for future support?

Reply:
Sorry I missed this. I don't know exactly - I'll look closer. Maybe it was placeholder in anticipation to bring `out-width` and `out-height` to Jupyter (as it is supported in **knitr** initially)